src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -2124,7 +2127,7 @@ bool registrationsQuery
       continue;
     }
     docs++;
-    LM_T(LmtMongo, ("retrieved document [%d]: '%s'", docs, r.toString().c_str()));
+    LM_T(LmtForward, ("retrieved document [%d]: '%s'", docs, r.toString().c_str()));
```

Comment:
We have several "retrieved document"-like traces, all them in the LmtMongo level. Let's keep it, pls.

Reply:
LmtMongo ...
It's extremely wide.
I get too much debugging if I turn it on.
OK for you to have both?

Reply:
But the other way arround, if tracelvel is change, then we will remove an important log message for these users just looking for LmtMongo.

Having both could be a good trade off.

Reply:
Fixed in 3dc4bcf83f00f7937ce79cacd25417575f5beaed (both traces)